### PR TITLE
Begin work on documentation improvements

### PR DESCRIPTION
**Pull request by jdegoes**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,7 +30,7 @@ The package strives to deliver on the following design goals:
 
 ### Legal
 
-Copyright (C) 2017-2018 John A. De Goes. All rights reserved.
+Copyright (C) 2017-2018 John A. De Goes and the ZIO Contributors. All rights reserved.
```
Comment (regiskuckaertz) [inline README.md]: 😎

File: core/shared/src/main/scala/scalaz/zio/ZIO.scala [hunk 2]
```diff
@@ -0,0 +1,18 @@
+package scalaz.zio
+
+/**
+ * A type class that allows people to use ZIO-specific code with
+ * other effect monads, so long as they can lift an arbitrary `IO`
+ * computation into their type. See the `interop` modules for
+ * instances of this type class for other effect types.
+ */
+trait ZIO[F[_, _]] {
+  def liftZIO[E, A](io: IO[E, A]): F[E, A]
```
Comment (regiskuckaertz) [inline core/shared/src/main/scala/scalaz/zio/ZIO.scala]: Closes #365

File: microsite/src/main/tut/index.md [hunk 3]
```diff
@@ -26,13 +29,15 @@ println(s"""libraryDependencies += "org.scalaz" %% "scalaz-zio" % "${scalaz.zio.
...
- * **Composability**. Purely functional code can't be combined with impure code that has side-effects without the straightforward reasoning properties of functional programming. `IO` lets you wrap up all effects into a purely functional package that lets you build composable real world programs.
- * **Concurrency**. `IO` has all the concurrency features of `Future`, and more, based on a clean fiber concurrency model designed to scale well past the limits of native threads. Unlike other effect monads, `IO`'s concurrency primitives do not leak threads.
+ * **Composability**. Purely functional code can't be combined with impure code that has side-effects without sacrificing the straightforward reasoning properties of functional programming. `IO` lets you wrap up all effects into a purely functional package that lets you build composable real world programs.
+ * **Concurrency**. `IO` has all the concurrency features of `Future`, and more, based on a clean fiber concurrency model designed to scale well past the limits of native threads. Unlike other effect monads, `IO`'s concurrency primitives do not leak resources.
```
Comment (regiskuckaertz) [inline microsite/src/main/tut/index.md]: > Unlike other effect monads

There may be a more positive way to phrase that (except if you intend to use a competitive tone in the copywriting).

Comment (jdegoes) [reply microsite/src/main/tut/index.md]: I agree. I will pull out this whole section, rewrite it (it’s very old), and put it in a new page about IO. Doesn’t belong here but didn’t want to do too much in one PR.

**Pull request merged**